frontend/packages/dev-console/src/components/import/image-search/ImageSearch.tsx

```diff
@@ -72,6 +76,11 @@ const ImageSearch: React.FC = () => {
       });
   };
 
+  const handleSave = (name: string) => {
+    setNewImageSecret(name);
+    values.searchTerm && handleSearch(values.searchTerm);
```

Comment:
what if `handleSearch` fails or couldn't search

Reply:
You mean if the user creates a secret with incorrect credentials? 
This is how it shows for incorrect credentials

![Screenshot from 2019-08-01 00-59-17](https://user-images.githubusercontent.com/22490998/62242119-1c536700-b3f8-11e9-84fa-0ae5a5f3ed1d.png)